### DISCUSSION TITLE
Which version of tools for OSX 10.13 ?

### DISCUSSION THREAD

**Discussion by larfran** [apprenticeharper/DeDRM_tools#2263] [General]
Hi, I have an old Mac, with OSX 10.13.
The version of Calibre is 3.48.
Wich version of DeDRM I should use for it ?
Thanks!

Edit: Ok, 6.8.0 is ok. Don't forget to unzip it and download from calibre app the files.zip into. That was my fault.

**Comment by ElleKayEm** [comment]
The latest 10.0.9 is meant to be backward compatible too.  But as you say, 6.8.x definitely works as long as you're dealing with older DRM types.
